test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -810,6 +819,7 @@ void UcxConnection::established(ucs_status_t status)
     _ucx_status = status;
     _context.remove_connection_inprogress(this);
     if (status == UCS_OK) {
+        print_addresses(false);
```

Comment:
what would be printed when passing false?

Reply:
actually nothing, I just reused this func to init addresses, need to refactor